### PR TITLE
Code review comments fixed

### PR DESCRIPTION
**Pull request by IraKutsa**
Problem with generated proto files imports

### CODE REVIEW

File: report_client/views.py [hunk 1]
```diff
@@ -18,10 +16,13 @@
...
-        response = stub.CreateReport(request=report)
-        json = MessageToJson(response, preserving_proto_field_name=True)
-        return Response(json, status=status.HTTP_201_CREATED)
+        try:
+            report = ParseDict(request.data, employee_report_pb2.Report())
+            response = stub.CreateReport(request=report)
+            json = MessageToJson(response, preserving_proto_field_name=True)
+            return Response(json, status=status.HTTP_201_CREATED)
+        except grpc.RpcError as e:
+            return Response(str(e), status=status.HTTP_500_INTERNAL_SERVER_ERROR)
```
Comment (OleksandrKotvytskyiVakoms) [inline report_client/views.py]: Why you don't want to return 400 error with error explanation? Raising 500 error to the external service not a good idea. Try to handle different error types.

File: report_server/server.py [hunk 2]
```diff
@@ -6,56 +7,59 @@
...
 from google.protobuf.json_format import MessageToDict, ParseDict
-from pymongo import MongoClient
+from pymongo import MongoClient, errors
 
-import employee_report_pb2_grpc, employee_report_pb2
+from protocol_buffers import employee_report_pb2_grpc
 
 
-def convert(report):
+def convert(report, message):
```
Comment (OleksandrKotvytskyiVakoms) [inline report_server/server.py]: maybe you should make this function as private or protected method of class?

File: report_server/server.py [hunk 3]
```diff
@@ -6,56 +7,59 @@
...
 
     def GetReportsByUser(self, request, context):
         logging.info('call GetReportsByUser')
         logging.info(request)
         reports = self.db.reports.find({"user_id": request.user_id})
-        by_user = employee_report_pb2.ReportsByUser(user_id=request.user_id)
+        by_user = employee_report_pb2_grpc.ReportsByUser(user_id=request.user_id)
         if reports:
-            by_user.reports.extend([convert(report) for report in reports])
+            by_user.reports.extend([convert(report, employee_report_pb2_grpc.Report()) for report in reports])
```
Comment (OleksandrKotvytskyiVakoms) [inline report_server/server.py]: `by_user.reports = list([convert(report, employee_report_pb2_grpc.Report()) for report in reports])`

or 

create a function convert_to_Report with partial function from functools lib, then use map builded_in function

code example:
```
from functool import partial

convert_to_Report = partial(convert, message=employee_report_pb2_grpc.Report())
```


so you will be able to use 
`list(map(convert_to_Report, reports))`

https://www.geeksforgeeks.org/python-map-function/
https://www.geeksforgeeks.org/partial-functions-python/

File: report_server/server.py [hunk 4]
```diff
@@ -6,56 +7,59 @@
...
         return by_user
 
     def GetReportsByProject(self, request, context):
         logging.info('call GetReportsByProject')
         logging.info(request)
         reports = self.db.reports.find({"project_id": request.project_id})
-        by_project = employee_report_pb2.ReportsByProject(project_id=request.project_id)
+        by_project = employee_report_pb2_grpc.ReportsByProject(project_id=request.project_id)
```
Comment (OleksandrKotvytskyiVakoms) [inline report_server/server.py]: same here

**Review by OleksandrKotvytskyiVakoms — CHANGES_REQUESTED** [review]
see the comments

File: report_server/server.py (hunk 2 above)
Comment (IraKutsa) [reply report_server/server.py]: why? PyCharm suggests to make it a function outside of the class

File: report_server/server.py (hunk 3 above)
Comment (IraKutsa) [reply report_server/server.py]: why do you think it's better?

File: report_server/server.py (hunk 2 above)
Comment (OleksandrKotvytskyiVakoms) [reply report_server/server.py]: But you didn't ask "why?" ?)

Comment (IraKutsa) [reply report_server/server.py]: because PyCharm suggests only now, when function becomes method :-)

**Comment by IraKutsa** [comment]
Importing issue is still present :-(

/home/kutsa/PycharmProjects/reports_django/venv/bin/python /home/kutsa/PycharmProjects/reports_django/report_server/server.py 
Traceback (most recent call last):
  File "/home/kutsa/PycharmProjects/reports_django/report_server/server.py", line 13, in <module>
    from protocol_buffers import employee_report_pb2_grpc, employee_report_pb2
  File "/home/kutsa/PycharmProjects/reports_django/protocol_buffers/employee_report_pb2_grpc.py", line 5, in <module>
    import employee_report_pb2 as employee__report__pb2
ModuleNotFoundError: No module named 'employee_report_pb2'

**Pull request merged**